server/sonar-server/src/main/java/org/sonar/server/permission/ws/GroupsAction.java

```diff
@@ -81,11 +81,16 @@ public void define(WebService.NewController context) {
         "<li>'Administer' rights on the specified project</li>" +
         "</ul>")
       .addPagingParams(DEFAULT_PAGE_SIZE, RESULTS_MAX_SIZE)
-      .addSearchQuery("sonar", "names").setDescription("Limit search to group names that contain the supplied string. Must have at least %d characters.<br/>" +
```

Comment:
You should not remove `addSearchQuery` but replace it by `createSearchQuery` in order to use `.setMinimumLength(SEARCH_QUERY_MIN_LENGTH)` on it

Reply:
done